### PR TITLE
net: initial import of ethertypes and the ethernet header

### PR DESCRIPTION
**Pull request by authmillenon**
Does what it says on the tin ;-)

### CODE REVIEW

File: sys/include/net/ng_ethertype.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#ifndef NG_ETHERTYPE_H_
+#define NG_ETHERTYPE_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* expand at will ;-) */
+#define NG_ETHERTYPE_IP     (0x0800)    /* IP */
```
Comment (OlegHahm) [inline sys/include/net/ng_ethertype.h]: protocol version 4

Comment (authmillenon) [inline sys/include/net/ng_ethertype.h]: Fixed

**Comment by authmillenon** [comment]
I also added some frame length definitions ;-)

**Comment by cgundogan** [comment]
Looks good to me. Travis is complaining about doxygen warnings, however.

**Comment by authmillenon** [comment]
Fixed. :-)

**Comment by authmillenon** [comment]
+ added a reserved ethertype ;-)

**Comment by cgundogan** [comment]
ACK still holds. Squash needed. When travis verifies then GO

**Comment by authmillenon** [comment]
Squashed.

**Pull request merged**